.github/workflows/cd.yml

```diff
@@ -33,6 +33,18 @@ jobs:
         with:
           python-version: '3.11'
 
+      -
+        name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '19.5.0' #TODO: check
+
+      -
+        name: Build Web Application
+        run: |
+          cd src/atopile/viewer
```

Comment:
It'd be better to use the `working-directory` option instead of the `cd` command https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory

(Since otherwise you'll need to `cd` back out again to not affect downstream steps)